rosidl_generator_py/resource/_msg.py.template

```diff
@@ -19,6 +28,11 @@ class @(spec.base_type.type)__Constants(type):
 @[for constant in spec.constants]@
             '@(constant.name)': @constant_value_to_py(constant.type, constant.value),
 @[end for]@
+@[for field in spec.fields]@
+@[if field.default_value]@
+            '@(field.name)__default': @value_to_py(field.type, field.default_value),
```

Comment:
I've updated the branch to use `@(field.name.upper())__DEFAULT` and also that arrays that have default values are turned into `tuple`s so they are immutable.